static/app/components/group/issueReplayCount.tsx

```diff
@@ -27,18 +27,23 @@ function IssueReplayCount({groupId}: Props) {
   const countDisplay = count > 50 ? '50+' : count;
 
   return (
-    <Tooltip title={t('This issue has %s replays available to view', countDisplay)}>
+    <StyledTooltip title={t('This issue has %s replays available to view', countDisplay)}>
       <ReplayCountLink
         to={`/organizations/${organization.slug}/issues/${groupId}/replays/`}
         aria-label="replay-count"
       >
         <IconPlay size="xs" />
         {countDisplay}
       </ReplayCountLink>
-    </Tooltip>
+    </StyledTooltip>
   );
 }
 
+const StyledTooltip = styled(Tooltip)`
+  display: flex;
+  align-items: center;
```

Comment:
tweak so the replay icon doesn't add extra height to rows.

<img width="454" alt="Screen Shot 2022-11-17 at 8 00 26 PM" src="https://user-images.githubusercontent.com/187460/202613878-7482fae8-204e-4f03-b585-8d8b6c3ef832.png">

Reply:
if you use skipWrapper on the tooltip it stops rendering a span, not sure if this is a hack for that